include/tvm/relax/nested_msg.h

```diff
@@ -370,6 +370,31 @@ void DecomposeNestedMsg(Expr expr, NestedMsg<T> msg, FType fvisitleaf) {
   }
 }
 
+template <typename T, typename FType>
+Expr ExprMutateFromNestedMsg(Expr expr, NestedMsg<T> msg, FType fvisitleaf) {
```

Comment:
How about RemapTupleLeaf?

Reply:
I prefer the word mutate. How about MutateTupleLeaf?

Reply:
How about TransformTupleLeaf? Mutate might make it sounds a bit inplace but it is also fine. Add a bit docs in public header

Reply:
I think this function tries to mutate in place as much as possible, which is the same as ExprMutator.

Reply:
sounds good, you can choose the right now, think a bit more i think MutateTupleLeaf and TransformTupleLeaf is fine. Indeed we should explain very carefully that it expands the var through TupleGetItem